File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md

```diff
@@ -0,0 +1,171 @@
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word Document to PDF in AWS Lambda
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in AWS Lambda**.
+
+## Steps to convert Word document to PDF in AWS Lambda
+
+Step 1: Create a new **AWS Lambda project** as follows.
+![AWS Lambda project](AWS_Images/Lambda_Images/Project-Template-WordtoPDF.png)
+
+Step 2: Select Blueprint as Empty Function and click **Finish**.
+![Select Blueprint as Empty Function](AWS_Images/Lambda_Images/Blueprint-AWS-WordtoPDF.png)
+
+Step 3: Install the following **Nuget packages** in your application from [Nuget.org](https://www.nuget.org/).
+
+* [Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core) 
+* [SkiaSharp.NativeAssets.Linux v2.88.2](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.88.2)
+
+![Install Syncfusion.DocIORenderer.Net.Core Nuget Package](Azure_Images/App_Service_Linux/Syncfusion_Nuget_Package_WordtoPDF.png)
+![Install SkiaSharp.NativeAssets.Linux Nuget Package](Azure_Images/App_Service_Linux/SkiaSharp_Nuget-Package_WordtoPDF.png)
+
+Step 4: Create a folder and copy the required data files and include the files to the project.
+![Create a folder](AWS_Images/Lambda_Images/Data-Folder-WordtoPDF.png)
+
+Step 5: Set the **copy to output directory** to **Copy if newer** to all the data files.
+![Property change for data files](AWS_Images/Lambda_Images/Property-WordtoPDF.png)
+
+Step 6: Include the following namespaces in **Function.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.Pdf;
+using Syncfusion.Drawing;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+step 7: Add the following code snippet in **Function.cs** to convert a Word document to PDF.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+/// <summary>
+/// A simple function that takes a string and does a ToUpper
+/// </summary>
+/// <param name="input"></param>
+/// <param name="context"></param>
+/// <returns></returns>
+public string FunctionHandler(string input, ILambdaContext context)
+{
+    string filePath = Path.GetFullPath(@"Data/Adventure.docx");
+    
+    //Load the file from the disk
+    FileStream fileStream = new FileStream(filePath, FileMode.Open, FileAccess.Read);
+ 
+    WordDocument document = new WordDocument(fileStream, FormatType.Docx); 
+    DocIORenderer render = new DocIORenderer();
+    PdfDocument pdf = render.ConvertToPDF(document);
+ 
+    //Save the document into stream
+    MemoryStream stream = new MemoryStream();
+
+    //Save the PDF document  
+    pdf.Save(stream);
+    document.Close();
```

Comment:
//Releases all resources used by the Word document and DocIO Renderer objects

//Closes the PDF document

Reply:
updated